doc/reference/reference_lua/net_box.rst

```diff
@@ -820,9 +828,39 @@ With the ``net.box`` module, you can use the following
                                           be replaced by trigger-function
     :return: nil or function pointer
 
+..  _net_box-on_shutdown:
+
+..  function:: conn:on_shutdown([trigger-function[, old-trigger-function]])
+
+    Define a trigger for shutdown when a :ref:`box.shutdown <system-events_box-shutdown>` event is received.
+
+    The trigger starts in a new fiber.
+    While the ``on_shutdown()`` trigger is running, the connection stays active.
+    It means that it is allowed to send new requests from a trigger callback.
+
+    After the trigger return, the ``net.box`` connection goes to the ``graceful_shutdown`` state
+    (check :ref:`the state diagram <net_box-options>` for details).
```

Comment:
When I click the link, I don't see the image and I have to scroll down. Consider inserting an anchor right before the diagram.

Reply:
Fixed